src/components/dialogs/network-modifications/static-var-compensator/creation/static-var-compensator-creation-dialog.tsx

```diff
@@ -0,0 +1,442 @@
+                    : null,
+                staticCompensator[CHARACTERISTICS_CHOICE] === CHARACTERISTICS_CHOICES.SUSCEPTANCE.id
+                    ? staticCompensator[MIN_SUSCEPTANCE]
+                    : null,
+                staticCompensator[CHARACTERISTICS_CHOICE] === CHARACTERISTICS_CHOICES.Q_AT_NOMINAL_V.id
+                    ? staticCompensator[MAX_Q_AT_NOMINAL_V]
+                    : null,
+                staticCompensator[CHARACTERISTICS_CHOICE] === CHARACTERISTICS_CHOICES.Q_AT_NOMINAL_V.id
+                    ? staticCompensator[MIN_Q_AT_NOMINAL_V]
+                    : null,
+                staticCompensator[VOLTAGE_REGULATION_MODE],
+                staticCompensator[VOLTAGE_SET_POINT],
+                staticCompensator[REACTIVE_POWER_SET_POINT],
+                staticCompensator[VOLTAGE_REGULATION_TYPE],
+                isDistantRegulation ? staticCompensator[EQUIPMENT]?.id : null,
+                isDistantRegulation ? staticCompensator[EQUIPMENT]?.type : null,
+                isDistantRegulation ? staticCompensator[VOLTAGE_LEVEL]?.id : null,
+                staticCompensator[ADD_STAND_BY_AUTOMATON],
+                staticCompensator[STAND_BY_AUTOMATON],
+                staticCompensator[ADD_STAND_BY_AUTOMATON] ? staticCompensator[LOW_VOLTAGE_SET_POINT] : null,
+                staticCompensator[ADD_STAND_BY_AUTOMATON] ? staticCompensator[HIGH_VOLTAGE_SET_POINT] : null,
+                staticCompensator[ADD_STAND_BY_AUTOMATON] ? staticCompensator[LOW_VOLTAGE_THRESHOLD] : null,
+                staticCompensator[ADD_STAND_BY_AUTOMATON] ? staticCompensator[HIGH_VOLTAGE_THRESHOLD] : null,
+                staticCompensator[ADD_STAND_BY_AUTOMATON] ? staticCompensator[B0] : null,
+                staticCompensator[ADD_STAND_BY_AUTOMATON] ? staticCompensator[Q0] : null,
+                !!editData,
+                editData ? editData.uuid : undefined,
+                toModificationProperties(staticCompensator)
+            ).catch((error) => {
+                snackError({
+                    messageTxt: error.message,
+                    headerId: 'StaticVarCompensatorCreationError',
+                });
+            });
+        },
+        [currentNodeUuid, editData, snackError, studyUuid]
+    );
+
+    const clear = useCallback(() => {
+        reset(emptyFormData);
+    }, [reset]);
+
+    const open = useOpenShortWaitFetching({
+        isDataFetched:
+            !isUpdate || editDataFetchStatus === FetchStatus.SUCCEED || editDataFetchStatus === FetchStatus.FAILED,
+        delay: FORM_LOADING_DELAY,
+    });
+
+    const onValidationError = (errors: any) => {
+        console.log('=============================errors', errors);
+        let tabsInError = [];
+        if (errors?.[CONNECTIVITY] !== undefined) {
+            tabsInError.push(StaticVarCompensatorCreationDialogTab.CONNECTIVITY_TAB);
+        }
+        if (errors?.[SETPOINTS_LIMITS] !== undefined) {
```

Comment:
not present in schema so this tab is not shown in red when some inside fields is on error